plugins/FlashSales/src/EasyAbp.EShop.Plugins.FlashSales.Application/EasyAbp/EShop/Plugins/FlashSales/FlashSalePlans/FlashSalePlanCacheItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using EasyAbp.EShop.Plugins.FlashSales.FlashSalePlans.Dtos;
+
+namespace EasyAbp.EShop.Plugins.FlashSales.FlashSalePlans;
+
+[Serializable]
+public class FlashSalePlanCacheItem : FlashSalePlanDto
```

Comment:
Better to implement `IMultiTenant` since the ABP framework could support tenant-special cache providers future.